java/Nikiforov-Oleksandr/src/com/zoolatech/lecture3/tasks/_2/Validator.java

```diff
@@ -0,0 +1,44 @@
+package com.zoolatech.lecture3.tasks._2;
+
+public interface Validator {
+    boolean isValid(UserAccount userAccount);
+
+    default String getStringFieldValue(UserAccount userAccount, String fieldName) {
+        switch (fieldName) {
+            case ("firstName") -> {
+                return userAccount.getFirstName();
+            }
+            case ("lastName") -> {
```

Comment:
braces and return statement is not required if case block contains only a single statement